examples/trials/mnist-nas/config.yml

```diff
@@ -0,0 +1,20 @@
+authorName: default
+experimentName: example_mnist
+trialConcurrency: 1
+maxExecDuration: 1h
+maxTrialNum: 10
+#choice: local, remote, pai
+trainingServicePlatform: local
```

Comment:
Also I suggest to add some unit-test for the parser-search-space and gen-code function.

Reply:
There is already an unit-test for them, but we can add more.

Reply:
unit-test at where?

Reply:
https://github.com/microsoft/nni/tree/v0.8/tools/nni_annotation/testcase/mutable_layer_usercode